core/extension/spx_sprite_mgr.h

```diff
@@ -89,6 +89,10 @@ class SpxSpriteMgr : SpxBaseMgr {
 	void set_color(GdObj obj, GdColor color);
 	GdColor get_color(GdObj obj);
 
+
+	void set_mosaic(GdObj obj, GdFloat strength);
```

Comment:
Scratch offers various graphic effects (https://www.en.scratch-wiki.info/wiki/Graphic_Effect#List_of_Graphical_Effects). Is it planned to implement all of them eventually?